<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2021 Opencvdl Hw1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>1. Find Contour</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_1_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1.1 Draw Contour</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_1_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1.2 Count Rings</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>There are _ rings in img1.jpg</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_1_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>There are _ rings in img2.jpg</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>2. Camera Calibration</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2.1 Find Corner</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2.2 Find Intrinsic </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2.4 Find Distortion</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select image:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2.3 Find Extrinsic</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>410</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>2.5 Show Result</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>210</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>3. Augmented Reality</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3.2 Show Words Vertically</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>3.1 Show Words on Board</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>4. Stereo Disparity Map</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>4.1 Stereo Disparity Map</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
